app/Http/Controllers/KaryawanController.php

```diff
@@ -16,11 +17,14 @@ class KaryawanController extends Controller
      */
     public function index()
     {
-        $karyawan = Karyawan::paginate(10);
-        return response()->json([
+        $karyawanService = new KaryawanService();
```

Comment:
seharusnya definisikan di atas di dalam conctructer seperti ini

`
$karyawanService KaryawanService;

public function __construct($karyawanService) {
$this->karyawanService = $karywanService;
}
`

jadi nanti pas di gunakan seperti ini 
`$this->karyawanService->getAllKaryawan();`